codespell_lib/data/dictionary.txt

```diff
@@ -20439,6 +20442,7 @@ missunderstood->misunderstood
 missuse->misuse
 missused->misused
 missusing->misusing
+mistakingly->mistakenly
```

Comment:
Looks like it should be rare:
https://www.lexico.com/definition/mistakingly

Reply:
[c791666](https://github.com/codespell-project/codespell/pull/2410/commits/c7916664695cb623f756b4d86042f138b0e032f3)